config/default.yml

```diff
@@ -1478,6 +1478,11 @@ Lint/MissingCopEnableDirective:
   # .inf for any size
   MaximumRangeSize: .inf
 
+Lint/MixedRegexpCaptureTypes:
+  Description: 'Do not mix named captures and numbered captures in a Regexp literal.'
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide/pull/793 has been merged. Can you add the StyleGuide address?

```yaml
StyleGuide: '#do-not-mix-named-and-numbered-captures'
```